backend/controller/user.go

```diff
@@ -167,20 +188,21 @@ func GetUserCreds(w http.ResponseWriter, r *http.Request) {
 
 func DeleteUserCreds(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	err, uId := helpers.ParseInt(w, r, "userId")
+	userIds := struct {
+		Users []uint `json:"users"`
+	}{}
+
+	err := json.NewDecoder(r.Body).Decode(&userIds)
 	if err != nil {
-		return
+		http.Error(w, "Malformed request", http.StatusBadRequest)
 	}
-	res := db.DB.Unscoped().Select(clause.Associations).Where(models.Credential{UserId: uId}).Delete(&[]models.Credential{})
+
+	res := db.DB.Unscoped().Where("user_id IN ?", userIds.Users).Find(&models.Credential{}).Delete(&[]models.Credential{})
```

Comment:
did we stest this one multiple creds? I had an account with 3 and it only deleted one

Reply:
verified it only deleted 1/2 the second time